### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by TorecLuik** [Neubias-WG5/W_NucleiSegmentation-Cellpose#2]
Please add a LICENSE file to the workflows. Some have it, some don't.
In theory the code is not to be reused or adapted if you don't have a license.

E.g. https://github.com/Neubias-WG5/W_PixCla-PSPNet-GlaS has Apache 2.0

We want to be compatible with the biaflows/cytomine workflows, but that does require LICENSE at some point.

**Comment by volker-baecker** [comment]
Hi @TorecLuik,
generally I agree with you. But how does this apply to neuronal network like cellpose? Cellpose is licensed under BSD 3-Clause, so does this apply here or can you license a deep learning model (i.e. the weights after the training)? 
Best,
Volker

**Comment by TorecLuik** [comment]
Hey Volker,

This code does not change Cellpose code itself, it just uses the public models that it downloads from the internet, so that just keeps its own separate license. BSD-3 is also not copyleft, so it does not enforce a license on this code. If being really official, I guess you could add a BSD-3 license notice as well for CellPose.

But at least the wrapper, dockerfile and metadata file, etc., should have their own license so anybody can officially adapt them.

Torec

**Comment by volker-baecker** [comment]
Yes, right in this case. The dockerfile and descriptor file probably lack the originality necessary for a copyright and therefore they don't need a license (but to not complicate things we should probably put them under a license nevertheless). The wrapper is modified from cytomine but with the idea to reintegrate things as much as possible with cytomine. So I guess what we need to do is apply the Apache License 2.0 to everything, except to the image analysis workflows that already have their own licenses. For those we should make it clear under which license they are.
Best,
Volker
